packages/extras/src/lib/components/controls/TransformControls/TransformControls.svelte

```diff
@@ -22,14 +22,17 @@
   type $$Slots = TransformControlsSlots
 
   export let autoPauseOrbitControls: $$Props['autoPauseOrbitControls'] = true
+  export let autoPauseTrackballControls: $$Props['autoPauseTrackballControls'] = true
   export let object: $$Props['object'] = undefined
 
   const { camera, renderer, invalidate, scene } = useThrelte()
 
-  const { orbitControls } = useControlsContext()
+  const { orbitControls, trackballControls } = useControlsContext()
   const isDragging = writable(false)
   const useAutoPauseOrbitControls = writable(autoPauseOrbitControls) ?? true
```

Comment:
@ixxie @MattFerraro I wonder if in this case we should compress into a single variable. In the other case splitting is valuable due to types but here we could reduce cognitive complexity for the end user by changing this to:
```svelte
<TransformControls autoPauseControls />
```
which would pause any camera controls detected with the aforementioned context. That to me seems like a simpler API. It would also require more work in this PR and notifying people of a breaking change in the changeset, so no worries though if we'd want to push that to later.

Reply:
yeah I think that's a better API, but I'd prefer to push it to later if that's okay!